.remarkrc.yml

```diff
@@ -2,5 +2,9 @@
 plugins:
   validate-links:
   lint-no-dead-urls:
-    skipUrlPatterns: ['https://pypistats.org/packages/git-machete', 'https://github.com/r-ryantm']
+    skipUrlPatterns: [
+      '/https:\/\/img\.shields\.io/',
```

Comment:
Does `/.../`  match against the entire URL, or just any part of the URL? I assumed the latter?

Reply:
@PawelLipski Based in this docs i infer the latter: 

skipUrlPatterns {Array} - Array of String | RegExp. A list of patterns for URLs that should be skipped. Each URL will be tested against each pattern, and will be ignored if new RegExp(pattern).test(url) === true. For example, with skipUrls: [/^http:\/\/(.*)url-to-ignore\.com/, 'https://never-check.com'], links with the URLs http://www.url-to-ignore.com/foo and https://never-check.com/foo/bar will not be checked.